mp_grow.c

```diff
@@ -8,15 +8,21 @@ mp_err mp_grow(mp_int *a, int size)
 {
    /* if the alloc size is smaller alloc more ram */
    if (a->alloc < size) {
+      mp_digit *dp;
+
+      if (size > MP_MAX_DIGIT_COUNT) {
+         return MP_MEM;
```

Comment:
Ah, not yet merged, good.
Would you do me a favour and use a dedicated error--code instead of `MP_MEM` for it, here and at `mp_init_size`? Maybe `MP_SIZE` or something in that line?
Not out of principle, I can make use of it.

Reply:
Good idea! I added that. What is your use case, may I ask?